Resources/Prototypes/Objectives/objectiveGroups.yml

```
@@ -32,7 +32,8 @@
 - type: weightedRandom
   id: TraitorObjectiveGroupKill
   weights:
-    KillRandomPersonObjective: 1
+    #KillRandomPersonObjective: 1 # DeltaV- Replaced for Teach Lesson
```

proper comment formatting jfc
```suggestion
    # KillRandomPersonObjective: 1 # DeltaV - Replaced for Teach Lesson
```